website/src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -23,7 +23,7 @@ const LanguageSelector = () => {
 
   // Memo the set of locales and their display names.
   const localesAndNames = useMemo(() => {
-    return router.locales.map((locale) => ({
+    return router?.locales?.map((locale) => ({
```

Comment:
why would the `router` be undefined?

Reply:
Since we are using this in a story it's easier to add this rather than try to mock the router and locale for storybook to render the Header component.

Reply:
Understandable, but this mixes testing logic with business logic.

we already have a router mock https://github.com/LAION-AI/Open-Assistant/blob/main/website/src/test-utils/createMockRouter.ts feel free to extend / modify / copy as necessary.

you can use `RouterProvider` for providing the mock, similar to the session.

Reply:
Ok, I'll take a look at using the mock router.